tests/desktop/developer_hub/test_submit_fxos_add_on.py

```diff
@@ -0,0 +1,65 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import os
+import shutil
+import uuid
+from zipfile import ZipFile
+
+import pytest
+
+from pages.desktop.developer_hub.content_tools import ContentTools
+from tests.base_test import BaseTest
+
+
+class TestSubmitFxOSAddOn(BaseTest):
+
+    @pytest.fixture
+    def fxos_add_on_name(self, request, mozwebqa):
```

Comment:
I'd just call this `add_on`, and perhaps return a dictionary with the name and path to the temporary zip file.